### ISSUE TITLE
Markdown syntax for internal linking is not supported by Structurizr

### ISSUE THREAD

**Issue by sirhaug** [structurizr/structurizr#98]
When writing documentation for Structurizr there are occations when I would like to link between files and systems, typically `... for more information, see [here](/docs/othermodule/file-to-link.md)`. I have yet to find a way to make this type of linking work without resorting to Structurizr specific syntax.

---

An example folder structure

```text
/ # Project root
|-- module1/
|   |-- intro.md
|   |-- foo.md
|   |-- bar.md
|-- module2/
|   |-- intro.md
|   |-- foofoo.md
|   |-- foobar.md
|-- workspace.dsl
```

where `workspace.dsl` looks like

```
workspace "Example workspace" {
    model {
        firstModule = softwareSystem "Module 1" "Some description" {
            !docs module1
        }
        secondModule = softwareSystem "Module 2" "Some other description" {
            !docs module2
        }
    }
}
```

---

In a GitHub based documentation repository, I could have the file `/module1/foo.md` contain the following text

```markdown
Lorem ipsum dolor sit amet, [consectetur](../module2/foobar.md) adipiscing elit.
```

or even link directly to a header in that same file.

With Structiruzr Lite, the resulting link on the rendered page for _Module 2_ brings up an error message when I click on it.

---

To get any internal link to work I have to use the _model name_ for each module.\
For example, as `softwareSystem "Module 2"` is defined in `workspace.dsl` I have to write
```markdown
Lorem ipsum dolor sit amet, [consectetur](Module%202) adipiscing elit.
```

to get the linking to work.

The problem with this is that it's not directly compatible with generic Markdown, which is a huge bummer.

---

So, to summarize:

- Is there another way of writing links so that they are directly compatible with Markdown and Structurizer,
- or is it possible to add support for generic links, maybe so that Structurizr converts internal Markdown links into its own format on render?

**Comment by simonbrowndotje** [comment]
Structurizr Lite has been updated, and should now better resolve filename based links between the pages/sections within a single software system. It still won't be able to resolve links between software systems though, such as transforming `[consectetur](../module2/foobar.md)` to `[consectetur](Module%202)`.

The general [usage recommendation](https://structurizr.com/help/usage-recommendations) is that a workspace contains the diagrams/documentation for a single software system. If you do want to have multiple software systems documented in a single workspace, you could use a [DSL plugin](https://github.com/structurizr/dsl/blob/master/docs/language-reference.md#plugins) to rewrite the links ... for example, iterate over the documentation sections and replace all occurrences of `../module2` with `Module%202`.
